os/Directories.mk

```diff
@@ -187,6 +187,15 @@ else
 KERNDEPDIRS += $(FSDIRS)
 endif
 
+
+# Add Kernel_tc Module
+
+ifeq ($(CONFIG_EXAMPLES_TESTCASE_KERNEL),y)
```

Comment:
CONFIG_EXAMPLES_TESTCASE_KERNEL has T.C.s only for os/kernel and lib/libc. That doesn't have T.Cs for driver, audio and so on. So, if you want add whole things inside this framework, you should modify config and name.